CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 * Use path.join to allow for windows development
 * Removed sauce labs config and install java 8 in a different way for travis ci
 * Added webdriver.io testing utilities
+* Update nightwatch to use docker
```

Comment:
Is this technically non-passive?  Assuming that users of this have to have docker installed?

Reply:
its non-passive from an environment standpoint but passive from a development one (after they install docker)

Reply:
Should we add that to the ReadMe?

Reply:
meh, they'll figure it out... .j/k